inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ieIRV7Tests)
```

Comment:
We need to update the CI because it's new target

Reply:
this target will not be commited